src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    padding-top: 40px;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    padding-top: 5px;
+    color: #616070;
+  }
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 17px;
+  }
+
+  &__stars_reviews {
+    display: flex;
+    align-items: center;
+    font-weight: 400;
```

Comment:
it's a default value
```suggestion
```

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/222682978-e7843d0e-8751-4f7c-8f88-59745b944fe2.png)